### DISCUSSION TITLE
Out of Sight status

### DISCUSSION THREAD

**Discussion by spamwax** [rgd87/Aptechka#61] [Q&A]
I cannot make this to work!
Is this for actual "line of sight" or is it a range check?

**Comment by d87** [comment]
Sorry it was broken, this thing relies on blizzard error message code and it sometimes changes and it's hard to notice
https://github.com/rgd87/Aptechka/releases/tag/9.0.21-beta2
Yeah, it's line of sight, like if someone is around the corner and your heals fail

**Reply by spamwax** [reply]
So for it to work do I need to cast a spell to trigger the error code, or it will automatically show all party members that are LOS'ing me?

**Reply by d87** [reply]
You need to cast something. Just showing is impossible
